src/Sql/Sql.sqlproj

```diff
@@ -442,6 +402,9 @@
     <Build Include="dbo\Views\SsoUserView.sql" />
     <Build Include="dbo\Views\TransactionView.sql" />
     <Build Include="dbo\Views\UserView.sql" />
+    <Build Include="dbo\Stored Procedures\Organization_ReadByLicenseKey.sql" />
+    <Build Include="dbo\Stored Procedures\Organization_ReadSelfHostedDetailsById.sql" />
+    <Build Include="dbo\Stored Procedures\OrganizationUser_ReadOccupiedSeatCountByOrganizationId.sql" />
```

Comment:
Why were these moved? Consider sorting the whole itemgroup.

Reply:
They were at the bottom of the file and they didn't seem to make sense to me underneath the Secrets Manager and Vault files, so I moved them above them but I'll move them further up with the other stored procedures